rerun_py/docs/gen_common_index.py

```diff
@@ -44,7 +44,7 @@ def all_archetypes() -> list[str]:
     quoted_strings = []
 
     # Regular expression pattern to match quoted strings
-    pattern = r'"([^"]*)"'
+    pattern = r'"(archetype.[^"]*)"'
```

Comment:
I think this is wrong - there are nothing in `archetypes/__init__.py` matching that